adminer/index.php

```diff
@@ -10,6 +10,16 @@ class PronamicAdminer extends Adminer {
 		public function name() {
 			return 'Pronamic Adminer';
 		}
+
+		/**
+		 * Login.
+		 *
+		 * @link https://www.adminer.org/en/password/
+		 * @link https://github.com/vrana/adminer/blob/7247f801bd06e51347d7ea671484e0fa6a883cbb/adminer/include/adminer.inc.php#L142-L152
+		 */
+		public function login( $login, $password ) {
+			return true;
```

Comment:
This poses a security issue, as this change removes the default passwordless login restriction. We only want to allow logging into SQLite databases.

```suggestion
			if ( defined( 'DRIVER' ) && 'sqlite' === DRIVER ) {
				return true;
			}

			return parent::login( $login, $password );
```

Reply:
I had already considered this, but left it out for simplicity. Added for now, but doesn't necessarily eliminate the security issue.

Reply:
> but doesn't necessarily eliminate the security issue.

It prevents passworldless login to e.g. a MySQL server; as is the case by default. If you're running a SQLite database in production, then the issue persists indeed.